Dockerfile

```diff
@@ -57,3 +57,7 @@ CMD ["wine64_anyuser", "msconvert" ]
 
 ADD mywine /usr/bin/
 RUN chmod ugo+rx /usr/bin/mywine
+
+# Fixes for running as non-root user
+RUN mkdir -p /mywineprefix && chmod 777 /mywineprefix
+RUN mkdir -p /net && chmod 777 /net
```

Comment:
Why can't the first line be in the mywine script? And why does /net need to be created and chmod'd?

Reply:
Perhaps a better way would be to have mywine put MYWINEPREFIX in a directory in /tmp? Or somewhere that any running user should be able to create a user in.